data/human/human missions.txt

```diff
@@ -9341,3 +9341,1306 @@ mission "Timothy Radrickson 7: Grave"
+			`	Tavish clears his throat, and then says quietly, "New India is a very recently colonized world. It is filled with life, and the remnants of life that have yet to be discovered."`
+			`	He leans in and says even more quietly, "On a recent expedition Kormak and I discovered what we believe to be remains of a crashed ship. A xeno-ship."`
+			`	"Aliens," Kormak adds helpfully.`
+			`	Tavish straightens up, "That is what we're after, and why we need your help! So you see you really are on the cutting edge of discovery with us. A fellow adventurer and explorer!"`
+			`	"Soggy," Kormak affirms.`
+				goto questions2
+			
+			label equipment1
+			action
+				set "equipment1"
+			`	"The deep jungles of New India are filled with aggressive tree-analogs." Tavish explains.`
+			`	"Monsters," Kormak says with bared teeth.`
+			`	"Unlike our more civilized trees on Earth, these ones are significantly more mobile. They will steal and smother anything left out, including pack animals and people."`
+			`	Kormak removes his cap and says solemnly, "Gerald."`
+			`	Tavish nods somberly, "Indeed, poor Gerald. If only his legs were as quick as his mind."`
+			`	"More cardio," Kormak agrees.`
+			`	"We had a special mechanical ring made to surround our campsite in ignited heat and prevent the tree-analog's tendrils from entering or interfering with our work."`
+			`	"Firewall," Kormak adds.`
+			`	"There was also a drill and sifter specially designed and programmed by Dr. Gerald to help uncover the vessel without damaging it," Tavish continues, "and, lastly, an underground sonar headset that would allow us to see its buried structure and avoid injuring it during our excavation."`
+			`	Tavish shrugs. "There were other things like tents, and picks, and shovels and what not. But all of that we can get on world. The ring, drill, and headset are all irreplaceable and absolutely necessary for this dig."`
+			`	"Irreplaceable," Kormak confirms.`
+				goto questions2
+
+			label end
+			action
+				clear "who1"
+				clear "india1"
+				clear "equipment1"
+			`	Tavish brightens as he says, "I knew an erudite fellow such as yourself would see the value of our work. And for now on we will check with you before we do anything too untoward, right, Kormak?"`
+			`	The red-haired giant nods, "Ask first."`
+			`	"Grand," Tavish claps. "Now, let's see what we can discover at <planet>."`
+				accept
+
+	on visit
+		dialog `You land on <planet>, but realize that Kormak and Tavish are on one of your escorts who hasn't entered the system yet. Better depart and wait for them.`
+	on complete
+		conversation
+			`The foam surf crashes against the island only a few short meters from where your ship sets down. The thundering waves make it hard to hear anything as you step outside. If you're going to have any hope of finding Hans Fuller and the stolen equipment your best bet will be the spaceport. Hopefully the sea will be a little quieter there.`
+
+
+mission "Tavish and Kormak 3"
```

Comment:
It's half past midnight, I'll do more... sometime soon hopefully. Here's where I stopped.